pkg/packagekit/updates.scss

```diff
@@ -6,6 +6,13 @@
     @extend .ct-card;
 }
 
+@media screen and (max-width: 1200px) {
```

Comment:
Do we do this somewhere else as well? Seems like it should be some generic pattern?

Reply:
This is copied from cockpit-podman, I am not sure if we use this somewhere else, if so we should maybe consider putting it in patternfly-cockpit.scss?

Reply:
AFAICS, pkg/lib/patternfly/patternfly-4-overrides.scss if it can be considered a bug in PF itself -- then it should be reported there, and added with a reference to the upstream issue. Otherwise, if it's cockpit specific, then pkg/lib/page.scss.

Reply:
Both of these cases occur in our own ListingTable, so maybe the fix should be there.

Reply:
We do have media queries all over the place... but do you mean if we should use variables instead of hardcoding the pixel* value?

(* pixels in CSS != pixels on your screen... sometimes they do, but it's different for high DPI devices, browser scaling, etc.)

For badge stretching, yeah, I agree with @martinpitt: This should probably be in our overrides and fixed upstream.

Reply:
A LabelGroup does not allow stretching of it's contents, so I went with that as solution and this CSS rule can be dropped.